crates/fluvio/src/config/tls.rs

```diff
@@ -122,18 +123,38 @@ pub struct TlsCerts {
     pub ca_cert: String,
 }
 
+impl Debug for TlsCerts {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        write!(f, "TlsCerts {{ domain: {} }}", self.domain)
+    }
+}
+
 impl TlsCerts {
     /// Attempts to write the inline TLS certs into temporary files
     ///
     /// Returns a `TlsPaths` populated with the paths where the
     /// temporary files were written.
     pub fn try_into_temp_files(&self) -> Result<TlsPaths, IoError> {
         use std::fs::write;
-        let tmp = std::env::temp_dir();
```

Comment:
Is there a reason for writing a cert in the client rather than installer?

Reply:
It was probably done to keep it in the impl of TlsCerts, but it makes more sense to have this is installer. Will fix